### PR TITLE
fix(admin-ui): environment variables imports

### PR DESCRIPTION
**Pull request by frdomovic**
# fix(admin-ui): environment variables imports

## Summary
Before it was throwing [this](https://github.com/kulshekhar/ts-jest/issues/3888) issue but fixed now
Screenshot 2024-05-26 at 12 14 19

Fixes # C20-73

## Summary by CodeRabbit

- **New Features**
    - Added `vite-plugin-environment` dependency to improve environment management.

- **Bug Fixes**
    - Updated network environment access in `AddRelease.tsx` and `PublishApplication.tsx` for better functionality.

- **Refactor**
    - Updated network assignment and added `NetworkId` import in `PublishApplication.tsx`.

- **Chores**
    - Included `EnvironmentPlugin` from 'vite-plugin-environment' in Vite configuration for enhanced plugin functionality.

### CODE REVIEW

File: node-ui/src/pages/PublishApplication.tsx [hunk 1]
```diff
@@ -97,8 +98,7 @@ export default function PublishApplication() {
 
   const addWalletAccount = async () => {
     const selector = await setupWalletSelector({
-      // @ts-ignore: The 'import.meta' meta-property is only allowed when the '--module' option ...
-      network: import.meta.env.VITE_NEAR_ENVIRONMENT ?? "testnet",
+      network: (process.env["VITE_NEAR_ENVIRONMENT"] as NetworkId) ?? "testnet",
```
[the remark this answers is not in the document]
Comment (frdomovic) [reply node-ui/src/pages/PublishApplication.tsx]: updated

File: node-ui/src/pages/AddRelease.tsx [hunk 2]
```diff
@@ -82,8 +83,7 @@ export default function AddRelease() {
 
   const addWalletAccount = async () => {
     const selector = await setupWalletSelector({
-      // @ts-ignore: The 'import.meta' meta-property is only allowed when the '--module' option ...
-      network: import.meta.env.VITE_NEAR_ENVIRONMENT ?? "testnet",
+      network: (process.env["VITE_NEAR_ENVIRONMENT"] as NetworkId) ?? "testnet",
```
[the remark this answers is not in the document]
Comment (frdomovic) [reply node-ui/src/pages/AddRelease.tsx]: updated

File: node-ui/src/pages/AddRelease.tsx [hunk 3]
```diff
@@ -83,7 +83,7 @@ export default function AddRelease() {
 
   const addWalletAccount = async () => {
     const selector = await setupWalletSelector({
-      network: (process.env["VITE_NEAR_ENVIRONMENT"] as NetworkId) ?? "testnet",
+      network: process.env["VITE_NEAR_ENVIRONMENT"] as NetworkId || "testnet",
```
Comment (MatejVukosav) [inline node-ui/src/pages/AddRelease.tsx]: Is missing env evaluated as null or undefined?

Comment (frdomovic) [reply node-ui/src/pages/AddRelease.tsx]: reverted to ?? , missing env returns undefined.
Screenshot 2024-05-28 at 15 12 09

Comment (MatejVukosav) [reply node-ui/src/pages/AddRelease.tsx]: Then ?? should be used. 👍

**Pull request merged**